tests/unit/fixtures/profiles.py

```diff
@@ -0,0 +1,175 @@
+import pytest
+from dbt.config import RuntimeConfig
+
+from tests.unit.utils import config_from_parts_or_dicts
+
+
+@pytest.fixture(scope="session", autouse=True)
+def base_project_cfg():
+    return {
+        "name": "X",
+        "version": "0.1",
+        "profile": "test",
+        "project-root": "/tmp/dbt/does-not-exist",
+        "quoting": {
+            "identifier": False,
+            "schema": False,
+        },
+        "config-version": 2,
+    }
+
+
+@pytest.fixture(scope="session", autouse=True)
+def target_http(base_project_cfg) -> RuntimeConfig:
```

Comment:
I'd consider dropping this type annotation just to drop the dependency on `dbt-core` in this file. At some point we'll want to do that anyway, so maybe we just do it then.